build.gradle

```diff
@@ -154,18 +154,14 @@ task startDatastore {
 }
 
 startDatastore << {
-    final String scriptExtension;
-    final String pathPrefix;
     if (IS_WINDOWS) {
-        scriptExtension = ".bat"
-        pathPrefix = ""
+        final command = "${project.rootDir.absolutePath}\\script\\start-datastore.bat"
+        println command
+        command.execute()
     } else {
-        scriptExtension = ".sh"
-        pathPrefix = "."
+        // Run datastore emulator
+        ".${File.separatorChar}script${File.separatorChar}start-datastore.sh".execute()
```

Comment:
Why not have the same code for Windows and Unix?
The only difference is the extension of the script. That's what you set depending on `IS_WINDOWS`.